build-logic/project-plugins/src/main/kotlin/com.hedera.hashgraph.root.gradle.kts

```diff
@@ -35,8 +35,6 @@ configureLifecycleTask("clean")
 
 configureLifecycleTask("assemble")
 
-configureLifecycleTask("check")
```

Comment:
Otherwise the CI pipeline call:

```
gradlew -p hedera-node check --continue --scan
```

will run the checks in `platform-sdk` tests again. For which we already have a separate stage. This is, because `hedera-node` includes `platform-sdk` now.

We may revise this at a later point by adjusting the GH action setup.

This is currently wrong on `develop` as I missed this in #7875

Reply:
We can revert this change as I will be revamping the CI pipeline end today to eliminate the duplicate execution.

Reply:
I think the revert causes trouble with the parallel execution again. Because the _platform_ tests also cannot run in parallel with the _node_ tests. I think this is a resources issue. Some tests assume they have the whole machine available.

See my other comment: https://github.com/hashgraph/hedera-services/pull/8352#discussion_r1314972756

Until that is resolved in the tests, I think we should consider to still keep two separate `check` runs on CI (as we have now).

This is to move forward with this PR now. I think ultimately the tests themselves need to be improved. Or the problematic tests need to be isolated to somehow run them in a special stage.

Reply:
Now that the pipeline is more integrated, I think we can also solve this in Gradle. I will have another look today.